draft-ietf-httpbis-digest-headers.md

```diff
@@ -1022,29 +907,29 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 Content-Encoding: br
 Content-Location: /items/123
-Digest: sha-256=4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=,
-        sha-512=pxo7aYzcGI88pnDnoSmAnaOEVys0MABhgvHY9+VI+ElE6
-                0jBCwnMPyA/s3NF3ZO5oIWA7lf8ukk+\n5KJzm3p5og==
+Representation-Digest: sha-256=:4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=:,
+        sha-512=:pxo7aYzcGI88pnDnoSmAnaOEVys0MABhgvHY9+VI+ElE6
+                0jBCwnMPyA/s3NF3ZO5oIWA7lf8ukk+\n5KJzm3p5og==:
 
 iwiAeyJoZWxsbyI6ICJ3b3JsZCJ9Aw==
```

Comment:
```suggestion

iwiAeyJoZWxsbyI6ICJ3b3JsZCJ9Aw==
```

typo

Reply:
what is the typo here? it looks identical to me

Reply:
Yes, my mistake.